src/rpcq.lisp

```diff
@@ -253,8 +252,8 @@ We distinguish between the following options for any field type:
     ;; handle defined message types
     ((symbolp field-type)
      (if required
```

Comment:
@ecpeterson @stylewarning I think this is a bug, but it has been in the code since the public release, so I want your opinion before I merge.

Reply:
This looks to me too like it was a bug.